src/sentry/models/team.py

```diff
@@ -164,6 +164,9 @@ class Meta:
 
     __repr__ = sane_repr("name", "slug")
 
+    def class_name(self):
```

Comment:
Is there a reason we don't use `type(self).__name__`?  Just curious it's not really a big issue.